test/integration/local/test_serving.py

```diff
@@ -87,14 +97,22 @@ def test_serving_calls_model_fn_once(image_uri, sagemaker_local_session, instanc
 
 
 @contextmanager
-def _predictor(model_tar, script, image, sagemaker_local_session, instance_type,
-               model_server_workers=None):
-    model = PyTorchModel('file://{}'.format(model_tar),
-                         ROLE,
-                         script,
-                         image=image,
-                         sagemaker_session=sagemaker_local_session,
-                         model_server_workers=model_server_workers)
+def _predictor(model_tar, script, image, sagemaker_local_session, instance_type, model_server_workers=None):
+    if 'gpu' in image:
+        env_vars = {
+            'NCCL_SHM_DISABLE': '1'
+        }
+    else:
+        env_vars = None
```

Comment:
Nit: `env_vars` can default to `None`, assigned only if the condition holds true, instead of the else block.

Reply:
Acknowledged. I will send out a new commit with this change.